server.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
+
+const http         = require('http');
+const finalhandler = require('finalhandler');
+const Router       = require('router');
+
+const router = new Router();
+
+router.get('/', (request, response) => {
+  // A good place to start!
+  response.end();
+});
+
+const server = http.createServer((request, response) => {
+  router(request, response, finalhandler(request, response));
+});
+
+exports.listen = function(port, callback) {
```

Comment:
Okay, do you have a sense of what I can relate it to in terms of stuff they already may know?